components/layout/GetStartedStep.tsx

```diff
@@ -19,6 +20,7 @@ export default function GetStartedStep({
   ctaText,
   ctaLink,
   children,
+  ariaLabel,
```

Comment:
```suggestion
  ariaLabels,
```
Can i give a little suggestion here? 
Thinking about legibility, when we just say `ariaLabel` we cant say from what, right?
Could be from Title, SectionContainer, etc.

So what do you think about, insted of using `ariaLabel`, use `ariaLabels` as an Object type which inside define from which one.

At this case could be something like `type ariaLabels = { ctaButton: string }`
Basically the same you did at `ShareButtons`.

There are other places in this PR that we can use the same concept to change.
Let me know if this make any sense :smiley:

Reply:
that's a great suggestion @LeoFuna! I think I've applied it to all the parts of the code where this applies. Please let me know if I missed anything. There are some elements where I just directly apply the aria-labels, not through props, and those remain unchanged

Reply:
Seems pretty good! Loved it 👏